### PR TITLE
Small bug fixes to improve UX.

### PR DESCRIPTION
**Pull request by charlesmaddock**
The photo editor now has a default crop. 
Made it easier for users to change their background color.
Increased a post's word limit until it renders faded slightly.

### CODE REVIEW

File: src/prod_components/agorize.js [hunk 1]
```diff
@@ -195,13 +195,13 @@ function Agorize({
 				        	<Input.TextArea
 				        		name="body"
 										disabled={!authorized}
+										autosize={{minRows: 4}}
 				        		placeholder={
 				        			agoragramType !== "comment" ?
-				        				"Text (icke-obligatoriskt)\nDetta är markdown, tryck på frågetecknet för mer information. "
+				        				"Text (icke-obligatoriskt)\nDetta är react-markdown, tryck på frågetecknet för mer information. "
```
Comment (Nautman) [inline src/prod_components/agorize.js]: React-markdown is the package that compiles markdown to react. There should be little difference between this compiler and other compilers.

Therefore I think it's more correct to say it's markdown. It is also how GitHub refers to it. 
image

However, this is such a small problem I don't think this needs to be fixed now. The component will eventually be deprecated in favour of [draft.js](https://github.com/facebook/draft-js).

File: src/prod_components/userEditingForm.js [hunk 2]
```diff
@@ -69,7 +69,8 @@ class UserEditingForm extends React.Component {
   }
 
   handleColourChange = color => {
-    this.props.setUserColour( color.hex )
+    // Removing live colour update for the moment since it cause issues
+    // this.props.setUserColour( color.hex )
```
Comment (Nautman) [inline src/prod_components/userEditingForm.js]: I would appreciate more information on what issues it causes :)

I think you've done the right choice disabling it for now. Maybe create a GitHub issue to track the problem and implement a solution?

File: src/prod_components/PhotoEditor.js [hunk 3]
```diff
@@ -14,7 +14,16 @@ const PhotoEditor = forwardRef((props, ref) => {
...
+  const [crop, setCrop] = useState({
+    aspect: 1 / 1,
+    unit: '%',
+    width: 80,
+    height: 80,
+    x: 10,
+    y: 10,
+    maxWidth: 1000,
+    minWidth: 60
+  })
```
Comment (Nautman) [inline src/prod_components/PhotoEditor.js]: I suppose this is the initial crop area?

Please reply to this comment or explain in the code what it is. Then I can approve this change 👍

**Comment by charlesmaddock** [comment]
Agree on the first point, however, there seems to be several things you
can’t do in react-markdown that you can do in regular markdown, such as
creating Images and tables. I didn’t want to mislead any experienced users
of markdown.

As for the second point, the issue is that setColor() updates a color value
that is set in the parent component, which in turn sets the value of the
color picker component, creating a loop where users cannot change their
color. This problem was previously fixed by calling setColor() once the
setting was done, but the problem was reintroduced, hence my minor edit. I
could add a git issue if you insist, however I doubt it it would be
necessary since the issue is quite small.

Finally, yes, that is a initial crop.

**Comment by kelszo** [comment]
Is this ready to be pulled? @Nautman @charlesmaddock

**Comment by charlesmaddock** [comment]
> Is this ready to be pulled? @Nautman @charlesmaddock

I would say so, although the change from "markdown" to "react-markdown" is unnecessary. However, as Douglas pointed out, that problem doesn't matter too much, since it will be changed soon anyway.

**Pull request merged**